public/javascripts/tinymce.editor_box.js

```diff
@@ -131,23 +131,32 @@ define([
     var equella_button = INST && INST.equellaEnabled ? ",instructure_equella" : "";
     instructure_buttons = instructure_buttons + equella_button;
 
-    var buttons1 = "bold,italic,underline,forecolor,backcolor,removeformat,justifyleft,justifycenter,justifyright,bullist,outdent,indent,sup,sub,numlist,table,instructure_links,unlink" + instructure_buttons + ",fontsizeselect,formatselect";
+    var buttons1 = "bold,italic,underline,forecolor,backcolor,removeformat,justifyleft,justifycenter,justifyright,bullist,outdent,indent,sup,sub,numlist,table,instructure_links,unlink" + instructure_buttons + ",formatselect,styleselect";
```

Comment:
On Wed, May 06, 2015 at 08:06:27PM -0700, Brian Sadler wrote:
> No clue what is going on here, but I just want to double check this is required?

It took a while to find that, but that string is what controls
the buttons on the wysiwyg editor. Each item in the comma-separated
list is a name of a button or drop down menu.

The Canvas authors have three versions for different screen widths
(determined at load time, so to see it, resize the browser then refresh)
to approximate responsiveness.

But what I did there was just remove the font size item and add a style
dropdown item in its place.